server/src/main.ts

```diff
@@ -4,12 +4,13 @@ import { NestFactory } from '@nestjs/core';
 import { NestExpressApplication } from '@nestjs/platform-express';
 import { resolve } from 'path';
 import { AppModule } from './app.module';
+import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
     const app = await NestFactory.create<NestExpressApplication>(AppModule);
     app.setGlobalPrefix('api');
     app.enableCors();
-
+    app.useGlobalPipes(new ValidationPipe())
```

Comment:
so this was needed in order to use the `IsNotEmpty` and other decorators?

Reply:
Yep!